### DISCUSSION TITLE
How to Turn ON and OFF a Relay in ESP8266 Board using MQTTX and MQTT Broker?

### DISCUSSION THREAD

**Discussion by awesomezed** [emqx/MQTTX#885] [Q&A] [answered]
**I want to simply turn ON and OFF a relay in the esp8266 dev board**, can someone provide me the **firmware and source code**. 
It will help me to build my project. I am expecting good advice from you.

Thank You.

**Comment by ysfscream** [comment, marked as answer]
Sorry for my late reply. Here is a tutorial blog about ESP8266, hope it can help you.

https://www.emqx.com/en/blog/esp8266_mqtt_led

**Reply by awesomezed** [reply]
Thank You SO MUCH
